common/changes/visualTests_2017-04-27-23-02.json

```diff
@@ -0,0 +1,25 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Regression Tests",
+      "type": "patch"
```

Comment:
Then you comments in the change file should reflect it :) Otherwise we'll have a note in the release note that says "Regression Tests".